src/Http/CakeHttpEventListener.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace CakeSentry\Http;
+
+use Cake\Event\EventListenerInterface;
+use Cake\Http\Client\ClientEvent;
+use Sentry\SentrySdk;
+use Sentry\Tracing\SpanContext;
+use Sentry\Tracing\SpanStatus;
+
+class CakeHttpEventListener implements EventListenerInterface
+{
+    /**
+     * @inheritDoc
+     */
+    public function implementedEvents(): array
+    {
+        return [
+            'HttpClient.afterSend' => 'handleAfterSend',
+        ];
+    }
+
+    /**
+     * @param \Cake\Http\Client\ClientEvent $event
+     * @return void
+     */
+    public function handleAfterSend(ClientEvent $event): void
+    {
+        $request = $event->getRequest();
+        $response = $event->getResult();
+
+        $parentSpan = SentrySdk::getCurrentHub()->getSpan();
+
+        if ($parentSpan !== null) {
+            $context = SpanContext::make()
+                ->setOp('http.client');
+            $span = $parentSpan->startChild($context);
+
+            // Set the span we just started as the current span
+            SentrySdk::getCurrentHub()->setSpan($span);
+
+            $uri = $request->getUri();
+            $fullUrl = $uri->getScheme() . '://' . $uri->getHost() . $uri->getPath() . '/';
+            $span
+                ->setDescription(sprintf('%s %s', $request->getMethod(), $fullUrl))
+                ->setStatus(SpanStatus::createFromHttpStatusCode($response?->getStatusCode() ?? 0))
+                ->setData([
+                    'http.request.method' => $request->getMethod(),
+                    'http.response.body.size' => $response?->getBody()?->getSize(),
+                    'http.response.status_code' => $response?->getStatusCode() ?? 0,
+                ])
```

Comment:
https://develop.sentry.dev/sdk/performance/modules/requests/ for all supported data attributes.